### DISCUSSION TITLE
The extension has been disabled by Google. Any fix ?

### DISCUSSION THREAD

**Discussion by PhilGoub** [openstyles/stylus#1859] [General]
On the extension page, Google says :

"This extension is no longer available because it doesn't follow best practices for Chrome extensions."

Any idea to fix this ?
Thx

**Comment by tophf** [comment]
We'll publish MV3 version soon, meanwhile either install [Stylus beta ](https://chromewebstore.google.com/detail/stylus-beta/apmmpaebfobifelkijhaljbmpcgbjbdo) or disable Google's BS by disabling these flags:

* `chrome://flags/#extension-manifest-v2-deprecation-unsupported` 
* `chrome://flags/#extension-manifest-v2-deprecation-warning`
* `chrome://flags/#extension-manifest-v2-deprecation-disabled`

**Reply by PhilGoub** [reply]
Thx a ton
